docs/control-structures.rst

```diff
@@ -705,9 +728,17 @@ The following example shows how to use an error string together with ``revert``
             );
             // Perform the purchase.
         }
+        function withdraw() public {
+            if (msg.sender != owner)
+                revert Unauthorized();
+
+            msg.sender.send(address(this).balance);
+        }
     }
 
-If you provide the reason string directly, then the two syntax options are equivalent, it is the developer's preference which one to use.
+The two ways ``if (!condition) revert(...);`` and ``require(condition, ...);`` are
+equivalent as long as the arguments to ``revert`` and ``require`` do not have side-effects,
+for example if they are just strings or simple error instances.
```

Comment:
About the error instance part: is that implemented now?

Reply:
Removed.